monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -530,11 +555,18 @@ def train(
             device_ids = tuple(range(num_gpus))
         logger.info(f"number of GPUs is {len(device_ids)}, device ids are {device_ids}")
         if len(device_ids) > 1:
-            self.train_parallel(configs=ensure_tuple(configs), device_ids=device_ids, **kwargs)
+            self.train_parallel(
+                configs=ensure_tuple(configs),
+                device_ids=device_ids,
+                trainer_class_name=self.trainer_class_name,
+                **kwargs,
+            )
         else:
             for cfg in ensure_tuple(configs):
                 for _fold in range(self.num_folds):
-                    self.train_single_model(config=cfg, fold=_fold, **kwargs)
+                    self.train_single_model(
+                        config=cfg, fold=_fold, trainer_class_name=self.trainer_class_name, **kwargs
```

Comment:
I think the `device_ids` should also be passed in case the user specify a GPU for single-GPU training. Thanks!

Reply:
I have incorporated an option within `train_single_model` here to facilitate support for `device_ids`.